chalk-solve/src/display.rs

```diff
@@ -25,7 +25,7 @@ pub use self::state::*;
 
 use self::utils::as_display;
 
-pub fn write_item<F, I, T>(f: &mut F, ws: &WriterState<'_, I>, v: &T) -> Result
+fn write_item<F, I, T>(f: &mut F, ws: &WriterState<'_, I>, v: &T) -> Result
```

Comment:
This is technically unrelated, but fairly insubstantial.

Reply:
Well, as long as `write_items` is still public and usable, this should be fine

Reply:
:+1:! I think it is a positive change, just that it probably should have ended up part of #558 rather than this PR.